docs/deployments/environments.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Environments / Instances
```

Comment:
if we are favouring using "instances" over "environments" in this guide, then let's lead with that for the titles

```suggestion
title: Instances / Environments
```